src/core/utilities/point/point.h

```diff
@@ -129,17 +130,21 @@ namespace core
         auto begin() { return r.begin(); }
         auto end() { return r.end(); }
 
+        auto const& operator()() const { return r; }
```

Comment:
is that one still needed?

Reply:
right you are, I thought I had removed it, one of the troubles on working on so many branches at once